routes/reviews-router.js

```diff
@@ -0,0 +1,16 @@
+const reviewsRouter = require("express").Router();
+const { getReviews } = require("../models/reviews");
+
+reviewsRouter
+    .route("/")
+    .get((req, res) => {
+        getReviews()
+        .then((reviews) => {
+            res.send({ reviews })
+
+        })
+        .catch(err => { console.log(err)})
```

Comment:
This catch is currently not really doing anything. We want to use .catch in conjunction with next, so either change it to go with the 500 server error or delete it